### DISCUSSION TITLE
Routing guard not hitting for flutter web

### DISCUSSION THREAD

**Discussion by vpdevportal** [nylo-core/nylo#157] [General] [answered]
When I'm on the Flutter web app If I hit the URL path of a particular page directly in a browser it doesn't hit the routing guard applied to that path.

**Comment by agordn52** [comment, marked as answer]
Hi @vpdevportal,

This is related to https://github.com/nylo-core/nylo/issues/118

At the moment Nylo can't run route guards on initialRoutes, you'll need to use a helper like `routeTo` to trigger it.
If you have ideas for a PR, let me know.

**Reply by vpdevportal** [reply]
👍
